src/test/components/MenuButtons.test.js

```diff
@@ -282,5 +282,38 @@ describe('app/MenuButtons', function() {
 
       expect(container.firstChild).toMatchSnapshot();
     });
+
+    it('with no statistics object should not make the app crash', async () => {
+      jest.useFakeTimers();
+      jest
+        .spyOn(Date.prototype, 'toLocaleString')
+        .mockImplementation(function() {
+          // eslint-disable-next-line babel/no-invalid-this
+          return 'toLocaleString ' + this.toUTCString();
+        });
+      // Using gecko profile because it has metadata and profilerOverhead data in it.
+      const profile = processProfile(createGeckoProfile());
+
+      // Differently, we are removing statistics objects from all overhead objects.
```

Comment:
nit: Not sure what `Differently` means here :)
Maybe:
`We are removing statistics objects from all overhead objects to test the robustness of our handling code.`

Reply:
Differently, compared to the previous tests that was related to overhead :)
Changed it now.